<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:media="http://search.yahoo.com/mrss/"
	
	>
<channel>
	<title>
	Comments on: Operads and the Tree of Life	</title>
	<atom:link href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/feed/" rel="self" type="application/rss+xml" />
	<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/</link>
	<description></description>
	<lastBuildDate>Mon, 21 Dec 2015 02:47:33 +0000</lastBuildDate>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>http://wordpress.com/</generator>
	<item>
		<title>
		By: Operads and Phylogenetic Trees &#124; Azimuth		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-74946</link>

		<dc:creator><![CDATA[Operads and Phylogenetic Trees &#124; Azimuth]]></dc:creator>
		<pubDate>Mon, 21 Dec 2015 02:47:33 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-74946</guid>

					<description><![CDATA[A few years ago, after hearing &lt;a href=&quot;http://statweb.stanford.edu/~susan/&quot; rel=&quot;nofollow&quot;&gt;Susan Holmes&lt;/a&gt; speak about &lt;a href=&quot;http://finmath.stanford.edu/~susan/papers/lap.pdf&quot; rel=&quot;nofollow&quot;&gt;the mathematics of phylogenetic trees&lt;/a&gt;, I became interested in their connection to algebraic topology.  I wrote an article about this here:

&#8226; John Baez, &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/&quot; rel=&quot;nofollow&quot;&gt;Operads and the tree of life&lt;/a&gt;, 6 July 2011.

In trying to the make the ideas precise I recruited the help of &lt;a href=&quot;https://www.maths.ox.ac.uk/people/nina.otter&quot; rel=&quot;nofollow&quot;&gt;Nina Otter&lt;/a&gt;, who was then a graduate student at ETH Z&#252;rich, and who is now a grad student at Oxford working on mathematical biology with &lt;a href=&quot;http://people.maths.ox.ac.uk/harrington/&quot; rel=&quot;nofollow&quot;&gt;Heather Harrington&lt;/a&gt;.

It took us quite a while to work out all the details, and I could never have done it myself.  But now we&#039;re done!  Here&#039;s our paper:

&#8226; John Baez and Nina Otter, &lt;a href=&quot;http://arxiv.org/abs/1512.03337&quot; rel=&quot;nofollow&quot;&gt;Operads and phylogenetic trees&lt;/a&gt;.]]></description>
			<content:encoded><![CDATA[<p>A few years ago, after hearing <a href="http://statweb.stanford.edu/~susan/" rel="nofollow">Susan Holmes</a> speak about <a href="http://finmath.stanford.edu/~susan/papers/lap.pdf" rel="nofollow">the mathematics of phylogenetic trees</a>, I became interested in their connection to algebraic topology.  I wrote an article about this here:</p>
<p>&bull; John Baez, <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/" rel="nofollow">Operads and the tree of life</a>, 6 July 2011.</p>
<p>In trying to the make the ideas precise I recruited the help of <a href="https://www.maths.ox.ac.uk/people/nina.otter" rel="nofollow">Nina Otter</a>, who was then a graduate student at ETH Z&uuml;rich, and who is now a grad student at Oxford working on mathematical biology with <a href="http://people.maths.ox.ac.uk/harrington/" rel="nofollow">Heather Harrington</a>.</p>
<p>It took us quite a while to work out all the details, and I could never have done it myself.  But now we&#8217;re done!  Here&#8217;s our paper:</p>
<p>&bull; John Baez and Nina Otter, <a href="http://arxiv.org/abs/1512.03337" rel="nofollow">Operads and phylogenetic trees</a>.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-36693</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Tue, 18 Feb 2014 12:14:23 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-36693</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-36664&quot;&gt;Mike Stay&lt;/a&gt;.

Wow, I&#039;m surprised you caught that now---this is a pretty old post!  I&#039;ve replaced the phylogenetic tree of doggies with a different one I happen to have.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-36664">Mike Stay</a>.</p>
<p>Wow, I&#8217;m surprised you caught that now&#8212;this is a pretty old post!  I&#8217;ve replaced the phylogenetic tree of doggies with a different one I happen to have.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Mike Stay		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-36664</link>

		<dc:creator><![CDATA[Mike Stay]]></dc:creator>
		<pubDate>Tue, 18 Feb 2014 00:11:43 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-36664</guid>

					<description><![CDATA[Thanks to budget cuts, the website with the canid phylogenetic tree is no longer available.  The Internet Archive managed to grab a &lt;a href=&quot;http://web.archive.org/web/20100501123735/http://www.nbii.gov/portal/server.pt/community/taxonomy,_phylogenetics___systematics/404/phylogenetic_trees/1617&quot; rel=&quot;nofollow&quot;&gt;copy&lt;/a&gt;, however.  You may want to update your post.]]></description>
			<content:encoded><![CDATA[<p>Thanks to budget cuts, the website with the canid phylogenetic tree is no longer available.  The Internet Archive managed to grab a <a href="http://web.archive.org/web/20100501123735/http://www.nbii.gov/portal/server.pt/community/taxonomy,_phylogenetics___systematics/404/phylogenetic_trees/1617" rel="nofollow">copy</a>, however.  You may want to update your post.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Phylogenetic Tree of Life &#171; Infinite Interior		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6754</link>

		<dc:creator><![CDATA[Phylogenetic Tree of Life &#171; Infinite Interior]]></dc:creator>
		<pubDate>Fri, 15 Jul 2011 07:45:07 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6754</guid>

					<description><![CDATA[[...] Operads and the Tree of Life « Azimuth. [...]]]></description>
			<content:encoded><![CDATA[<p>[&#8230;] Operads and the Tree of Life « Azimuth. [&#8230;]</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6726</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Thu, 14 Jul 2011 09:08:50 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6726</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6692&quot;&gt;John Baez&lt;/a&gt;.

Thanks for all that, Graham!  As you noticed, I don&#039;t even know the difference between phylogenetics and phylogenomics.  Well, I do now.  But I have a lot of catching up to do.  

By the way, when I gave my talk at UQAM the fellow in charge of the combinatorics seminar, Franco Saliola, said he had been to a nice talk by the mathematical biologist &lt;a href=&quot;http://math.berkeley.edu/~lpachter/publications.html&quot; rel=&quot;nofollow&quot;&gt;Lior Pachter&lt;/a&gt;.  Have you heard of him?  His website says:

&lt;blockquote&gt;
I work on the fundamental problem of comparative genomics: the determination of the origins and evolutionary history of the nucleotides in all extant genomes. My work incorporates various aspects of genomics, including the reconstruction of ancestral genomes (paleogenomics), the modeling of genome dynamics (phylogenomics and systems biology) and the assignment of function to genome elements (functional genomics and epigenomics).

In addition to working on algorithms and mathematical foundations for comparative genomics, I also work on genome projects and perform large scale computational analyses. I have been a member of the mouse, rat, chicken and fly genome sequencing consortia, and the ENCODE project.

My research draws on tools from discrete mathematics, algebra and statistics. I am also interested in questions in these subjects that are motivated by biology problems. 
&lt;/blockquote&gt;]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6692">John Baez</a>.</p>
<p>Thanks for all that, Graham!  As you noticed, I don&#8217;t even know the difference between phylogenetics and phylogenomics.  Well, I do now.  But I have a lot of catching up to do.  </p>
<p>By the way, when I gave my talk at UQAM the fellow in charge of the combinatorics seminar, Franco Saliola, said he had been to a nice talk by the mathematical biologist <a href="http://math.berkeley.edu/~lpachter/publications.html" rel="nofollow">Lior Pachter</a>.  Have you heard of him?  His website says:</p>
<blockquote><p>
I work on the fundamental problem of comparative genomics: the determination of the origins and evolutionary history of the nucleotides in all extant genomes. My work incorporates various aspects of genomics, including the reconstruction of ancestral genomes (paleogenomics), the modeling of genome dynamics (phylogenomics and systems biology) and the assignment of function to genome elements (functional genomics and epigenomics).</p>
<p>In addition to working on algorithms and mathematical foundations for comparative genomics, I also work on genome projects and perform large scale computational analyses. I have been a member of the mouse, rat, chicken and fly genome sequencing consortia, and the ENCODE project.</p>
<p>My research draws on tools from discrete mathematics, algebra and statistics. I am also interested in questions in these subjects that are motivated by biology problems.
</p></blockquote>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Graham		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6693</link>

		<dc:creator><![CDATA[Graham]]></dc:creator>
		<pubDate>Mon, 11 Jul 2011 19:07:51 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6693</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6692&quot;&gt;John Baez&lt;/a&gt;.

&quot;What branching patterns might be particularly likely and/or unlikely?&quot;

Trees have a topology and they can have branch lengths or node times. As far as the topology alone is concerned, the main observation is the phylogenetic trees are unbalanced. I have just added an image to

http://www.azimuthproject.org/azimuth/show/Tree+of+life

to show what I mean. This imbalance show up at every level down to trees with 4 leaves which is the smallest size that imbalance can occur. All the obvious mathematical models (eg a birth-death process assuming constant rates of birth and death) make more balanced trees than this.

As far as node times are concerned it will depend on context. For example, for gene trees within a species, coalescent theory
(http://en.wikipedia.org/wiki/Coalescent_theory) gives a useful model. Looking backward in time, the coalescences happen very quickly, then slow down. For a gene sampled from say 10 individuals, and assuming a constant population size, the expected time for the first coalescence is proportional to 1/(10*9), the additional time to the next coalescence is proportional to 1/(9*8), and so on down the the last pair to meet with expected time proportional to 1/(2*1). Looking forwards in time, this gives a tree that grows faster then exponential.

&quot;In other words, that you can work only with trees whose branches all make it to the present, without any harm. I&#039;ve been hoping to find some context in which this actually causes problems. Do you know about that?&quot;

It can cause problems if you want to estimate dates. For example, there are 22 Crocodylla (crocodiles, alligators and gharials), and they separated from the rest of the tree of life a very long time ago (lets say 100My for the sake of argument). If you assumed there were no extinctions, you would estimate the time of the most recent common ancestor of extant Crocodylla (that is, the time of the first speciation in the phylogenetic tree for Crocodylla ) to be a very long time ago as well. Unlike the coalescence case, the expected times go like 1/21, 1/20, ... 1/2, 1/1 going back in time, so - very roughly - you might estimate this time as 1/(1+1/2+...1/21) = 1/3.64 = .27 of 100My since the ancestor species separated from the rest of the tree of life, that is 73My ago. More realistically, there will have been many extinctions and quite likely there were once many more than 22 in this group. In this case, it could easily be that the most recent common ancestor of extant Crocodylla is very recent.

It also causes problems if you know some dates from fossils and want to estimate speciation and extinction rates. This article might be a good place to start. 

&quot;Estimating diversification rates from phylogenetic information&quot;. By Ricklefs R E, Trends Ecol Evol. 2007.
http://www.bio-nica.info/biblioteca/Ricklefs2007PhylogeneticInformation.pdf

Finally, &#039;phylogenetics&#039; means estimating trees from genes and &#039;phylogenomics&#039; means estimating trees from whole genomes.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6692">John Baez</a>.</p>
<p>&#8220;What branching patterns might be particularly likely and/or unlikely?&#8221;</p>
<p>Trees have a topology and they can have branch lengths or node times. As far as the topology alone is concerned, the main observation is the phylogenetic trees are unbalanced. I have just added an image to</p>
<p><a href="http://www.azimuthproject.org/azimuth/show/Tree+of+life" rel="nofollow ugc">http://www.azimuthproject.org/azimuth/show/Tree+of+life</a></p>
<p>to show what I mean. This imbalance show up at every level down to trees with 4 leaves which is the smallest size that imbalance can occur. All the obvious mathematical models (eg a birth-death process assuming constant rates of birth and death) make more balanced trees than this.</p>
<p>As far as node times are concerned it will depend on context. For example, for gene trees within a species, coalescent theory<br />
(<a href="http://en.wikipedia.org/wiki/Coalescent_theory" rel="nofollow ugc">http://en.wikipedia.org/wiki/Coalescent_theory</a>) gives a useful model. Looking backward in time, the coalescences happen very quickly, then slow down. For a gene sampled from say 10 individuals, and assuming a constant population size, the expected time for the first coalescence is proportional to 1/(10*9), the additional time to the next coalescence is proportional to 1/(9*8), and so on down the the last pair to meet with expected time proportional to 1/(2*1). Looking forwards in time, this gives a tree that grows faster then exponential.</p>
<p>&#8220;In other words, that you can work only with trees whose branches all make it to the present, without any harm. I&#8217;ve been hoping to find some context in which this actually causes problems. Do you know about that?&#8221;</p>
<p>It can cause problems if you want to estimate dates. For example, there are 22 Crocodylla (crocodiles, alligators and gharials), and they separated from the rest of the tree of life a very long time ago (lets say 100My for the sake of argument). If you assumed there were no extinctions, you would estimate the time of the most recent common ancestor of extant Crocodylla (that is, the time of the first speciation in the phylogenetic tree for Crocodylla ) to be a very long time ago as well. Unlike the coalescence case, the expected times go like 1/21, 1/20, &#8230; 1/2, 1/1 going back in time, so &#8211; very roughly &#8211; you might estimate this time as 1/(1+1/2+&#8230;1/21) = 1/3.64 = .27 of 100My since the ancestor species separated from the rest of the tree of life, that is 73My ago. More realistically, there will have been many extinctions and quite likely there were once many more than 22 in this group. In this case, it could easily be that the most recent common ancestor of extant Crocodylla is very recent.</p>
<p>It also causes problems if you know some dates from fossils and want to estimate speciation and extinction rates. This article might be a good place to start. </p>
<p>&#8220;Estimating diversification rates from phylogenetic information&#8221;. By Ricklefs R E, Trends Ecol Evol. 2007.<br />
<object data="http://www.bio-nica.info/biblioteca/Ricklefs2007PhylogeneticInformation.pdf" type="application/pdf" width="100%" height="800" style="height: 800px;"><p><a href="http://www.bio-nica.info/biblioteca/Ricklefs2007PhylogeneticInformation.pdf">Click to access Ricklefs2007PhylogeneticInformation.pdf</a></p></object></p>
<p>Finally, &#8216;phylogenetics&#8217; means estimating trees from genes and &#8216;phylogenomics&#8217; means estimating trees from whole genomes.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6692</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Mon, 11 Jul 2011 17:20:27 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6692</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6676&quot;&gt;John Baez&lt;/a&gt;.

Graham wrote:

&lt;blockquote&gt;
Actually biologists do assume a probability distribution on the branching patterns, and the choice can affect the results. You are right that little is known about speciation rates and extinction rates, so guesses have to be made. 
&lt;/blockquote&gt;

Thanks for the correction!  I only know a little about phylogenomics, so it&#039;s nice to know that when I say something wrong, there&#039;s a chance you&#039;ll appear and correct me. 

What branching patterns might be particularly likely and/or unlikely?

Also, you mention &#039;extinction rates&#039;.   One person told me that extinction events were completely ignored in the process of guessing a phylogenetic tree from present-day DNA data.  In other words, that you can work only with trees whose branches all make it to the present, without any harm.  I&#039;ve been hoping to find some context in which this actually causes problems.  Do you know about that?  

(Of course we need to think about extinction events when we also use DNA data from extinct species... but I&#039;m not talking about that now.)]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6676">John Baez</a>.</p>
<p>Graham wrote:</p>
<blockquote><p>
Actually biologists do assume a probability distribution on the branching patterns, and the choice can affect the results. You are right that little is known about speciation rates and extinction rates, so guesses have to be made.
</p></blockquote>
<p>Thanks for the correction!  I only know a little about phylogenomics, so it&#8217;s nice to know that when I say something wrong, there&#8217;s a chance you&#8217;ll appear and correct me. </p>
<p>What branching patterns might be particularly likely and/or unlikely?</p>
<p>Also, you mention &#8216;extinction rates&#8217;.   One person told me that extinction events were completely ignored in the process of guessing a phylogenetic tree from present-day DNA data.  In other words, that you can work only with trees whose branches all make it to the present, without any harm.  I&#8217;ve been hoping to find some context in which this actually causes problems.  Do you know about that?  </p>
<p>(Of course we need to think about extinction events when we also use DNA data from extinct species&#8230; but I&#8217;m not talking about that now.)</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: reperiendi		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6686</link>

		<dc:creator><![CDATA[reperiendi]]></dc:creator>
		<pubDate>Mon, 11 Jul 2011 04:47:36 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6686</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6615&quot;&gt;Tom Leinster&lt;/a&gt;.

Here&#039;s a nice article on it:
http://www.bunniestudios.com/blog/?p=353]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6615">Tom Leinster</a>.</p>
<p>Here&#8217;s a nice article on it:<br />
<a href="http://www.bunniestudios.com/blog/?p=353" rel="nofollow ugc">http://www.bunniestudios.com/blog/?p=353</a></p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Tom Leinster		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6685</link>

		<dc:creator><![CDATA[Tom Leinster]]></dc:creator>
		<pubDate>Mon, 11 Jul 2011 00:13:42 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6685</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6664&quot;&gt;John Baez&lt;/a&gt;.

John, the stuff you&#039;re doing prompted me to make a pdf version of my note available, since most people (including me) prefer pdf to ps these days.  It&#039;s here: &lt;a href=&quot;http://www.maths.ed.ac.uk/~tl/w.pdf&quot; rel=&quot;nofollow&quot;&gt;www.maths.ed.ac.uk/~tl/w.pdf&lt;/a&gt;.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6664">John Baez</a>.</p>
<p>John, the stuff you&#8217;re doing prompted me to make a pdf version of my note available, since most people (including me) prefer pdf to ps these days.  It&#8217;s here: <a href="http://www.maths.ed.ac.uk/~tl/w.pdf" rel="nofollow">http://www.maths.ed.ac.uk/~tl/w.pdf</a>.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Todd Trimble		</title>
		<link>https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6683</link>

		<dc:creator><![CDATA[Todd Trimble]]></dc:creator>
		<pubDate>Sun, 10 Jul 2011 19:39:05 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=4145#comment-6683</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6680&quot;&gt;Mike Shulman&lt;/a&gt;.

I was about to ask innocently, &quot;what&#039;s an HMM?&quot; Then I googled it and figured it referred to &lt;a href=&quot;http://en.wikipedia.org/wiki/Hidden_Markov_model&quot; rel=&quot;nofollow&quot;&gt;this&lt;/a&gt;. But I guess it wouldn&#039;t hurt to record this fact for the similarly innocent. :-)]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2011/07/06/operads-and-the-tree-of-life/#comment-6680">Mike Shulman</a>.</p>
<p>I was about to ask innocently, &#8220;what&#8217;s an HMM?&#8221; Then I googled it and figured it referred to <a href="http://en.wikipedia.org/wiki/Hidden_Markov_model" rel="nofollow">this</a>. But I guess it wouldn&#8217;t hurt to record this fact for the similarly innocent. :-)</p>
]]></content:encoded>
		
			</item>
	</channel>
</rss>
