<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:media="http://search.yahoo.com/mrss/"
	
	>
<channel>
	<title>
	Comments on: Operads and Phylogenetic Trees	</title>
	<atom:link href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/feed/" rel="self" type="application/rss+xml" />
	<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/</link>
	<description></description>
	<lastBuildDate>Sat, 02 Jan 2016 04:18:18 +0000</lastBuildDate>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>http://wordpress.com/</generator>
	<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75553</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Sat, 02 Jan 2016 04:18:18 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75553</guid>

					<description><![CDATA[There are some nice comments about operads and phylogenetic trees &lt;a href=&quot;https://golem.ph.utexas.edu/category/2015/12/operads_and_phylogenetic_trees.html#comments&quot; rel=&quot;nofollow&quot;&gt;over on the &lt;em&gt;n&lt;/em&gt;-Category Caf&#233;&lt;/a&gt;, including a nice approach to trees by Joachim Kock.]]></description>
			<content:encoded><![CDATA[<p>There are some nice comments about operads and phylogenetic trees <a href="https://golem.ph.utexas.edu/category/2015/12/operads_and_phylogenetic_trees.html#comments" rel="nofollow">over on the <em>n</em>-Category Caf&eacute;</a>, including a nice approach to trees by Joachim Kock.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75541</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Fri, 01 Jan 2016 22:47:23 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75541</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75540&quot;&gt;Gisser Martin&lt;/a&gt;.

I used unshuffles in my work on Lie 2-algebra, and one needs shuffles to &lt;a href=&quot;https://en.wikipedia.org/wiki/Tensor_algebra#Coalgebra_structures&quot; rel=&quot;nofollow&quot;&gt;make the tensor algebra of vector space into a bialgebra&lt;/a&gt;.  But I haven&#039;t been shuffling much lately.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75540">Gisser Martin</a>.</p>
<p>I used unshuffles in my work on Lie 2-algebra, and one needs shuffles to <a href="https://en.wikipedia.org/wiki/Tensor_algebra#Coalgebra_structures" rel="nofollow">make the tensor algebra of vector space into a bialgebra</a>.  But I haven&#8217;t been shuffling much lately.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Gisser Martin		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75540</link>

		<dc:creator><![CDATA[Gisser Martin]]></dc:creator>
		<pubDate>Fri, 01 Jan 2016 22:26:34 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75540</guid>

					<description><![CDATA[Last year I became a bit obsessed with the shuffle multiplication resp. unshuffle coproduct. Do you have any use for it?

It appears in many branches of maths incl. operads (Loday/Vallette: Algebraic Operads - of which I know nothing) and Markov chains (Persi Diaconis). But it has not been explicitly appreciated in tensor calculus (Veblen already used it implicitly) - which I plan to change...]]></description>
			<content:encoded><![CDATA[<p>Last year I became a bit obsessed with the shuffle multiplication resp. unshuffle coproduct. Do you have any use for it?</p>
<p>It appears in many branches of maths incl. operads (Loday/Vallette: Algebraic Operads &#8211; of which I know nothing) and Markov chains (Persi Diaconis). But it has not been explicitly appreciated in tensor calculus (Veblen already used it implicitly) &#8211; which I plan to change&#8230;</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: gisparents		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75498</link>

		<dc:creator><![CDATA[gisparents]]></dc:creator>
		<pubDate>Wed, 30 Dec 2015 21:40:24 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75498</guid>

					<description><![CDATA[Hi John, This is a wonderful work. I met Nina when she visited SJSU, a few weeks ago. I talked to her and also briefly talked about my work. I am working on the relationship between C-sets and phylogenetics. Look forward to hearing more. Regards, Shabeena]]></description>
			<content:encoded><![CDATA[<p>Hi John, This is a wonderful work. I met Nina when she visited SJSU, a few weeks ago. I talked to her and also briefly talked about my work. I am working on the relationship between C-sets and phylogenetics. Look forward to hearing more. Regards, Shabeena</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Graham		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75026</link>

		<dc:creator><![CDATA[Graham]]></dc:creator>
		<pubDate>Thu, 24 Dec 2015 10:41:18 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75026</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75009&quot;&gt;John Baez&lt;/a&gt;.

&lt;blockquote&gt;
  You said earlier that the phylogenetic trees you consider are more fancy than the ones we consider. Maybe if you explained them to me I could see if they were operations in a mathematically interesting operad. (I can’t promise to do anything more useful than that.)
&lt;/blockquote&gt;

That&#039;s fine, I don&#039;t have much hope this is going to lead anywhere very useful. It might give me a different way of looking at something.

I am working with the multispecies coalescent model. This model has a bunch of &#039;gene trees&#039; sitting inside a species tree. The individual trees are quite ordinary, but the gene trees are constrained by the species tree. There are some pictures (Figs 1 and 9) in this paper: http://mbe.oxfordjournals.org/content/27/3/570.full. The model incorporates a simple model from population genetics along each branch of the species tree.

The trees are binary and ultrametric. The tips of the species tree are labelled with species. I&#039;ll just talk about a single gene tree. They are all basically the same sort of thing, so if you understand one gene tree, you understand them all. The gene tree will have at least as many tips as the species tree, usually more. At the tips are DNA sequences which have been sampled from individuals belonging to one of the species.  The constraint is that going back in time, two DNA sequences cannot meet (coalesce) until the species they belong to have met.

A more biological description. Suppose you have two human DNA sequences for a particular gene. (You can generally get two sequences from each individual organism so they could be from the same person.) Suppose the human population is 50,000. That might sound crazy given the current population, but over the last six million years it is a reasonable guess. That means 100,000 copies in the human gene pool, and so using a very simple model of population genetics, the probability that this pair meets is 1/100,000 per generation, say 25 years. So the probability they have not met by six million years into the past is about exp(-6/2.5) or 9%. If they don&#039;t meet, it is an example of &#039;incomplete lineage sorting&#039;. Once they&#039;re back that far, they can meet DNA sequences from chimps. It means that a gene tree can have a different topology to the species tree.

I knew about the latex thing, and even put one &#039;latex&#039; into my post. Then I got distracted by something else, and clicked on post before adding the rest. &#039;infinitesimal stochastic&#039; is fine. Biologists just say &#039;rate matrix&#039;.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75009">John Baez</a>.</p>
<blockquote><p>
  You said earlier that the phylogenetic trees you consider are more fancy than the ones we consider. Maybe if you explained them to me I could see if they were operations in a mathematically interesting operad. (I can’t promise to do anything more useful than that.)
</p></blockquote>
<p>That&#8217;s fine, I don&#8217;t have much hope this is going to lead anywhere very useful. It might give me a different way of looking at something.</p>
<p>I am working with the multispecies coalescent model. This model has a bunch of &#8216;gene trees&#8217; sitting inside a species tree. The individual trees are quite ordinary, but the gene trees are constrained by the species tree. There are some pictures (Figs 1 and 9) in this paper: <a href="http://mbe.oxfordjournals.org/content/27/3/570.full" rel="nofollow ugc">http://mbe.oxfordjournals.org/content/27/3/570.full</a>. The model incorporates a simple model from population genetics along each branch of the species tree.</p>
<p>The trees are binary and ultrametric. The tips of the species tree are labelled with species. I&#8217;ll just talk about a single gene tree. They are all basically the same sort of thing, so if you understand one gene tree, you understand them all. The gene tree will have at least as many tips as the species tree, usually more. At the tips are DNA sequences which have been sampled from individuals belonging to one of the species.  The constraint is that going back in time, two DNA sequences cannot meet (coalesce) until the species they belong to have met.</p>
<p>A more biological description. Suppose you have two human DNA sequences for a particular gene. (You can generally get two sequences from each individual organism so they could be from the same person.) Suppose the human population is 50,000. That might sound crazy given the current population, but over the last six million years it is a reasonable guess. That means 100,000 copies in the human gene pool, and so using a very simple model of population genetics, the probability that this pair meets is 1/100,000 per generation, say 25 years. So the probability they have not met by six million years into the past is about exp(-6/2.5) or 9%. If they don&#8217;t meet, it is an example of &#8216;incomplete lineage sorting&#8217;. Once they&#8217;re back that far, they can meet DNA sequences from chimps. It means that a gene tree can have a different topology to the species tree.</p>
<p>I knew about the latex thing, and even put one &#8216;latex&#8217; into my post. Then I got distracted by something else, and clicked on post before adding the rest. &#8216;infinitesimal stochastic&#8217; is fine. Biologists just say &#8216;rate matrix&#8217;.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75010</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Wed, 23 Dec 2015 22:18:15 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75010</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75007&quot;&gt;Jack Morava&lt;/a&gt;.

Cool!  I&#039;ll check it out.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75007">Jack Morava</a>.</p>
<p>Cool!  I&#8217;ll check it out.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75009</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Wed, 23 Dec 2015 22:05:08 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75009</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75003&quot;&gt;Graham&lt;/a&gt;.

Thanks!

&lt;blockquote&gt;
  The main thing I learnt was that composability is the key thing about operads (whereas I was thinking vaguely of topology, metric, or measure).
&lt;/blockquote&gt;

Right.  An operad is a bunch of operations that you can compose.  I attempted to say this in the blog article:

&lt;blockquote&gt;
... trees are often used to describe the composition of &lt;i&gt;n&lt;/i&gt;-ary operations. This is formalized in the theory of &lt;a href=&quot;https://en.wikipedia.org/wiki/Operad_theory&quot; rel=&quot;nofollow&quot;&gt;operads&lt;/a&gt;. An &#039;operad&#039; is an algebraic stucture where for each natural number $latex n=0,1,2,\dots$ we have a set $latex O_n$ whose elements are considered as abstract &lt;i&gt;n&lt;/i&gt;-ary operations, not necessarily operating on anything yet. An element $latex f \in O_n$ can be depicted as a planar tree with one vertex and &lt;i&gt;n&lt;/i&gt; labelled leaves:

&lt;div align=&quot;center&quot;&gt;&lt;img src=&quot;http://math.ucr.edu/home/baez/ecological/phylo/corolla.jpg&quot; alt=&quot;&quot; /&gt;&lt;/div&gt;

We can compose these operations in a tree-like way to get new operations:

&lt;div align=&quot;center&quot;&gt;&lt;img src=&quot;http://math.ucr.edu/home/baez/ecological/phylo/operad_composition.jpg&quot; alt=&quot;&quot; /&gt;&lt;/div&gt;

and an associative law holds, making this sort of composite unambiguous:

&lt;div align=&quot;center&quot;&gt;&lt;img src=&quot;http://math.ucr.edu/home/baez/ecological/phylo/operad_associativity.jpg&quot; alt=&quot;&quot; /&gt;&lt;/div&gt;

&lt;/blockquote&gt;

The formula you mention here:

&lt;blockquote&gt;
  The formula for a single step in the calculation looks like this. Suppose a vertex $latex x$ in the tree has $latex r$ children, which have vectors $latex v_1, \dots, v_r \in \mathbb{R}^X$, and the branches between $latex x$ and its children have lengths $latex t_1, \dots, t_r$. Suppose $latex S$ is a $latex &#124;X&#124;$ by $latex &#124;X&#124;$ infinitesimal stochastic matrix which gives the rates at which members of $latex X$ turn into one another. Then we make $latex r$ vectors $latex \exp(S t_i) v_i$, and multiply these element-wise to make the vector at $latex x$.
&lt;/blockquote&gt;

is implicit in some stuff discussed in our paper, and even in the blog article here---though we were hiding it amid clouds of jargon, to make certains sorts of mathematician able to understand it better.  The idea, as we see it, is that the phylogenetic operad is built from two simpler ones:

&#8226; one called $latex [0,\infty)$ that describes time evolution, where the genome randomly changes according to a Markov process,

and

&#8226; one called $latex \mathrm{Com}$ that describes branching, where the genome gets duplicated, or &#039;&lt;i&gt;n&lt;/i&gt;-plicated&#039;.

Technically the phylogenetic operad is the &lt;a href=&quot;https://en.wikipedia.org/wiki/Coproduct&quot; rel=&quot;nofollow&quot;&gt;&#039;coproduct&#039;&lt;/a&gt; $latex \mathrm{Com} + [0,\infty)$ of these two other operads.

Of course I&#039;m not claiming this viewpoint is better for normal humans.

You said earlier that the phylogenetic trees &lt;i&gt;you&lt;/i&gt; consider are more fancy than the ones we consider.  Maybe if you explained them to me I could see if they were operations in a mathematically interesting operad.  (I can&#039;t promise to do anything more useful than that.)

By the way, I fixed your LaTeX: when you post a comment there are instructions directly above that say how to use LaTeX here, but almost nobody ever reads them.  In the process I took the liberty of changing &quot;stochastic&quot; to &quot;infinitesimal stochastic&quot;.  A &lt;b&gt;stochastic&lt;/b&gt; matrix is one that describes a finite amount of time evolution for a Markov process: namely, a square matrix of nonnegative numbers where the columns sum to 1.  We need some other name for a matrix $latex S$ such that $latex \exp(tS)$ is stochastic for all $latex t \ge 0$.  Some people call this an &lt;b&gt;intensity matrix&lt;/b&gt; or &lt;b&gt;stochastic Hamiltonian&lt;/b&gt;.  But I really want an adjective: I can&#039;t grammatically say &quot;now let us prove that this matrix is intensity&quot;.  Also, since in physics one speaks of the Hamiltonian of a particular physical system, one should not also use that word to denote a matrix with particular mathematical properties.

So, I use the adjective &lt;b&gt;infinitesimal stochastic&lt;/b&gt; to denote a matrix such that $latex \exp(tS)$ is stochastic for all $latex t \ge 0$.  This goes along with a tradition where we think of $latex S$ as describing an &#039;infinitesimal&#039; amount of time evolution, and $latex \exp(t S)$ as describing a &#039;finite&#039; amount of time evolution.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75003">Graham</a>.</p>
<p>Thanks!</p>
<blockquote><p>
  The main thing I learnt was that composability is the key thing about operads (whereas I was thinking vaguely of topology, metric, or measure).
</p></blockquote>
<p>Right.  An operad is a bunch of operations that you can compose.  I attempted to say this in the blog article:</p>
<blockquote><p>
&#8230; trees are often used to describe the composition of <i>n</i>-ary operations. This is formalized in the theory of <a href="https://en.wikipedia.org/wiki/Operad_theory" rel="nofollow">operads</a>. An &#8216;operad&#8217; is an algebraic stucture where for each natural number <img src="https://s0.wp.com/latex.php?latex=n%3D0%2C1%2C2%2C%5Cdots&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="n=0,1,2,&#92;dots" class="latex" /> we have a set <img src="https://s0.wp.com/latex.php?latex=O_n&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="O_n" class="latex" /> whose elements are considered as abstract <i>n</i>-ary operations, not necessarily operating on anything yet. An element <img src="https://s0.wp.com/latex.php?latex=f+%5Cin+O_n&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="f &#92;in O_n" class="latex" /> can be depicted as a planar tree with one vertex and <i>n</i> labelled leaves:</p>
<div align="center"><img src="https://i1.wp.com/math.ucr.edu/home/baez/ecological/phylo/corolla.jpg" alt="" /></div>
<p>We can compose these operations in a tree-like way to get new operations:</p>
<div align="center"><img src="https://i0.wp.com/math.ucr.edu/home/baez/ecological/phylo/operad_composition.jpg" alt="" /></div>
<p>and an associative law holds, making this sort of composite unambiguous:</p>
<div align="center"><img src="https://i0.wp.com/math.ucr.edu/home/baez/ecological/phylo/operad_associativity.jpg" alt="" /></div>
</blockquote>
<p>The formula you mention here:</p>
<blockquote><p>
  The formula for a single step in the calculation looks like this. Suppose a vertex <img src="https://s0.wp.com/latex.php?latex=x&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="x" class="latex" /> in the tree has <img src="https://s0.wp.com/latex.php?latex=r&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="r" class="latex" /> children, which have vectors <img src="https://s0.wp.com/latex.php?latex=v_1%2C+%5Cdots%2C+v_r+%5Cin+%5Cmathbb%7BR%7D%5EX&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="v_1, &#92;dots, v_r &#92;in &#92;mathbb{R}^X" class="latex" />, and the branches between <img src="https://s0.wp.com/latex.php?latex=x&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="x" class="latex" /> and its children have lengths <img src="https://s0.wp.com/latex.php?latex=t_1%2C+%5Cdots%2C+t_r&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="t_1, &#92;dots, t_r" class="latex" />. Suppose <img src="https://s0.wp.com/latex.php?latex=S&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="S" class="latex" /> is a <img src="https://s0.wp.com/latex.php?latex=%7CX%7C&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="|X|" class="latex" /> by <img src="https://s0.wp.com/latex.php?latex=%7CX%7C&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="|X|" class="latex" /> infinitesimal stochastic matrix which gives the rates at which members of <img src="https://s0.wp.com/latex.php?latex=X&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="X" class="latex" /> turn into one another. Then we make <img src="https://s0.wp.com/latex.php?latex=r&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="r" class="latex" /> vectors <img src="https://s0.wp.com/latex.php?latex=%5Cexp%28S+t_i%29+v_i&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;exp(S t_i) v_i" class="latex" />, and multiply these element-wise to make the vector at <img src="https://s0.wp.com/latex.php?latex=x&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="x" class="latex" />.
</p></blockquote>
<p>is implicit in some stuff discussed in our paper, and even in the blog article here&#8212;though we were hiding it amid clouds of jargon, to make certains sorts of mathematician able to understand it better.  The idea, as we see it, is that the phylogenetic operad is built from two simpler ones:</p>
<p>&bull; one called <img src="https://s0.wp.com/latex.php?latex=%5B0%2C%5Cinfty%29&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="[0,&#92;infty)" class="latex" /> that describes time evolution, where the genome randomly changes according to a Markov process,</p>
<p>and</p>
<p>&bull; one called <img src="https://s0.wp.com/latex.php?latex=%5Cmathrm%7BCom%7D&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;mathrm{Com}" class="latex" /> that describes branching, where the genome gets duplicated, or &#8216;<i>n</i>-plicated&#8217;.</p>
<p>Technically the phylogenetic operad is the <a href="https://en.wikipedia.org/wiki/Coproduct" rel="nofollow">&#8216;coproduct&#8217;</a> <img src="https://s0.wp.com/latex.php?latex=%5Cmathrm%7BCom%7D+%2B+%5B0%2C%5Cinfty%29&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;mathrm{Com} + [0,&#92;infty)" class="latex" /> of these two other operads.</p>
<p>Of course I&#8217;m not claiming this viewpoint is better for normal humans.</p>
<p>You said earlier that the phylogenetic trees <i>you</i> consider are more fancy than the ones we consider.  Maybe if you explained them to me I could see if they were operations in a mathematically interesting operad.  (I can&#8217;t promise to do anything more useful than that.)</p>
<p>By the way, I fixed your LaTeX: when you post a comment there are instructions directly above that say how to use LaTeX here, but almost nobody ever reads them.  In the process I took the liberty of changing &#8220;stochastic&#8221; to &#8220;infinitesimal stochastic&#8221;.  A <b>stochastic</b> matrix is one that describes a finite amount of time evolution for a Markov process: namely, a square matrix of nonnegative numbers where the columns sum to 1.  We need some other name for a matrix <img src="https://s0.wp.com/latex.php?latex=S&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="S" class="latex" /> such that <img src="https://s0.wp.com/latex.php?latex=%5Cexp%28tS%29&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;exp(tS)" class="latex" /> is stochastic for all <img src="https://s0.wp.com/latex.php?latex=t+%5Cge+0&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="t &#92;ge 0" class="latex" />.  Some people call this an <b>intensity matrix</b> or <b>stochastic Hamiltonian</b>.  But I really want an adjective: I can&#8217;t grammatically say &#8220;now let us prove that this matrix is intensity&#8221;.  Also, since in physics one speaks of the Hamiltonian of a particular physical system, one should not also use that word to denote a matrix with particular mathematical properties.</p>
<p>So, I use the adjective <b>infinitesimal stochastic</b> to denote a matrix such that <img src="https://s0.wp.com/latex.php?latex=%5Cexp%28tS%29&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;exp(tS)" class="latex" /> is stochastic for all <img src="https://s0.wp.com/latex.php?latex=t+%5Cge+0&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="t &#92;ge 0" class="latex" />.  This goes along with a tradition where we think of <img src="https://s0.wp.com/latex.php?latex=S&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="S" class="latex" /> as describing an &#8216;infinitesimal&#8217; amount of time evolution, and <img src="https://s0.wp.com/latex.php?latex=%5Cexp%28t+S%29&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;exp(t S)" class="latex" /> as describing a &#8216;finite&#8217; amount of time evolution.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Jack Morava		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75007</link>

		<dc:creator><![CDATA[Jack Morava]]></dc:creator>
		<pubDate>Wed, 23 Dec 2015 22:00:43 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75007</guid>

					<description><![CDATA[Dear John,

You might be interested in smooth blowup of the Billera-Holmes-Vogtmann model for tree space described in

&#8226; &lt;a href=&quot;http://www.sciencedirect.com/science/article/pii/S0196885815000378&quot; rel=&quot;nofollow&quot;&gt;Navigation in tree spaces&lt;/a&gt;, &lt;i&gt;Adv. Appl. Math.&lt;/i&gt; &lt;b&gt;67&lt;/b&gt; (2015), 75--95.

It&#039;s also connected to operads (more precisely, to the Deligne-Mumford-Knudsen operad defined by genus zero &lt;b&gt;real&lt;/b&gt; algebraic curves); its spaces are aspherical, with an interesting (system of) fundamental group(s), analogous in some ways to the operad defined by the braid groups. It has an intrinsic pseudometric, with wall-crossing phenomena that look suspiciously biological...

Best, sincerely

&lt;pre&gt;&lt;code&gt;                   (:+{)}  Jack
&lt;/code&gt;&lt;/pre&gt;]]></description>
			<content:encoded><![CDATA[<p>Dear John,</p>
<p>You might be interested in smooth blowup of the Billera-Holmes-Vogtmann model for tree space described in</p>
<p>&bull; <a href="http://www.sciencedirect.com/science/article/pii/S0196885815000378" rel="nofollow">Navigation in tree spaces</a>, <i>Adv. Appl. Math.</i> <b>67</b> (2015), 75&#8211;95.</p>
<p>It&#8217;s also connected to operads (more precisely, to the Deligne-Mumford-Knudsen operad defined by genus zero <b>real</b> algebraic curves); its spaces are aspherical, with an interesting (system of) fundamental group(s), analogous in some ways to the operad defined by the braid groups. It has an intrinsic pseudometric, with wall-crossing phenomena that look suspiciously biological&#8230;</p>
<p>Best, sincerely</p>
<pre><code>                   (:+{)}  Jack
</code></pre>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Graham		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-75003</link>

		<dc:creator><![CDATA[Graham]]></dc:creator>
		<pubDate>Wed, 23 Dec 2015 19:15:00 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-75003</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-74984&quot;&gt;John Baez&lt;/a&gt;.

Thanks, John. The main thing I learnt was that composability is the key thing about operads (whereas I was thinking vaguely of topology, metric, or measure).

&lt;blockquote&gt;
  A Markov model maps each tree to a matrix that describes how a probability distribution on the genome data of the species at the root of the tree stochastically evolves to a probability distributions on the genome data of the descendant species at the leaves. Ways of gluing trees together get sent to ways of combining these matrices.
  
  (I’m being quite rough here, just to get the point across with a minimum of distraction.)
  
  Of course this is not be how you usually think of a Markov model—like the Jukes–Cantor model,
&lt;/blockquote&gt;

Actually, that is very much how I think about phylogenetic analysis as a whole. Except, upside down and inside out.

We start with probability distributions on the genome data of the descendant species at the leaves. These are what we observe in the DNA of various present-day organisms.  (Usually these are regarded as known sequences, ie, as points rather than distributions, but that&#039;s for computational efficiency not realism.) Then using Felsenstein’s tree-pruning algorithm, we work recursively back to the root, calculating vectors in $latex \mathbb{R}^X.$ (The vectors are called &#039;conditional likelihoods of subtrees&#039; or &#039;partial likelihoods&#039;.) An extra step at the root converts the vector there to a number, which is the probability of the observed data, given the tree. In other words, it is the likelihood of the tree, given the observations. The likelihood of the tree, given the observations, is The Central Object of study in phylogenetics analysis (though I don&#039;t suppose many biologists would put it that way!).

The formula for a single step in the calculation looks like this. Suppose a vertex $latex x$ in the tree has $latex r$ children, which have vectors $latex v_1, \dots, v_r \in \mathbb{R}^X$, and the branches between $latex x$ and its children have lengths $latex t_1, \dots, t_r$. Suppose $latex S$ is a $latex &#124;X&#124;$ by $latex &#124;X&#124;$ infinitesimal stochastic matrix which gives the rates at which members of $latex X$ turn into one another. Then we make $latex r$ vectors $latex \exp(S t_i) v_i$, and multiply these element-wise to make the vector at $latex x$.

This seems the best bet for finding something that interests me into something &#039;operadic&#039;.

I already knew the Billera-Holmes-Vogtmann paper. And yes, that does seem about halfway between you and me. As I read it, their main reason why biologists should care is that it provides (in principle) ways of summarising The Central Object.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-74984">John Baez</a>.</p>
<p>Thanks, John. The main thing I learnt was that composability is the key thing about operads (whereas I was thinking vaguely of topology, metric, or measure).</p>
<blockquote><p>
  A Markov model maps each tree to a matrix that describes how a probability distribution on the genome data of the species at the root of the tree stochastically evolves to a probability distributions on the genome data of the descendant species at the leaves. Ways of gluing trees together get sent to ways of combining these matrices.</p>
<p>  (I’m being quite rough here, just to get the point across with a minimum of distraction.)</p>
<p>  Of course this is not be how you usually think of a Markov model—like the Jukes–Cantor model,
</p></blockquote>
<p>Actually, that is very much how I think about phylogenetic analysis as a whole. Except, upside down and inside out.</p>
<p>We start with probability distributions on the genome data of the descendant species at the leaves. These are what we observe in the DNA of various present-day organisms.  (Usually these are regarded as known sequences, ie, as points rather than distributions, but that&#8217;s for computational efficiency not realism.) Then using Felsenstein’s tree-pruning algorithm, we work recursively back to the root, calculating vectors in <img src="https://s0.wp.com/latex.php?latex=%5Cmathbb%7BR%7D%5EX.&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;mathbb{R}^X." class="latex" /> (The vectors are called &#8216;conditional likelihoods of subtrees&#8217; or &#8216;partial likelihoods&#8217;.) An extra step at the root converts the vector there to a number, which is the probability of the observed data, given the tree. In other words, it is the likelihood of the tree, given the observations. The likelihood of the tree, given the observations, is The Central Object of study in phylogenetics analysis (though I don&#8217;t suppose many biologists would put it that way!).</p>
<p>The formula for a single step in the calculation looks like this. Suppose a vertex <img src="https://s0.wp.com/latex.php?latex=x&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="x" class="latex" /> in the tree has <img src="https://s0.wp.com/latex.php?latex=r&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="r" class="latex" /> children, which have vectors <img src="https://s0.wp.com/latex.php?latex=v_1%2C+%5Cdots%2C+v_r+%5Cin+%5Cmathbb%7BR%7D%5EX&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="v_1, &#92;dots, v_r &#92;in &#92;mathbb{R}^X" class="latex" />, and the branches between <img src="https://s0.wp.com/latex.php?latex=x&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="x" class="latex" /> and its children have lengths <img src="https://s0.wp.com/latex.php?latex=t_1%2C+%5Cdots%2C+t_r&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="t_1, &#92;dots, t_r" class="latex" />. Suppose <img src="https://s0.wp.com/latex.php?latex=S&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="S" class="latex" /> is a <img src="https://s0.wp.com/latex.php?latex=%7CX%7C&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="|X|" class="latex" /> by <img src="https://s0.wp.com/latex.php?latex=%7CX%7C&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="|X|" class="latex" /> infinitesimal stochastic matrix which gives the rates at which members of <img src="https://s0.wp.com/latex.php?latex=X&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="X" class="latex" /> turn into one another. Then we make <img src="https://s0.wp.com/latex.php?latex=r&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="r" class="latex" /> vectors <img src="https://s0.wp.com/latex.php?latex=%5Cexp%28S+t_i%29+v_i&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="&#92;exp(S t_i) v_i" class="latex" />, and multiply these element-wise to make the vector at <img src="https://s0.wp.com/latex.php?latex=x&#038;bg=ffffff&#038;fg=333333&#038;s=0&#038;c=20201002" alt="x" class="latex" />.</p>
<p>This seems the best bet for finding something that interests me into something &#8216;operadic&#8217;.</p>
<p>I already knew the Billera-Holmes-Vogtmann paper. And yes, that does seem about halfway between you and me. As I read it, their main reason why biologists should care is that it provides (in principle) ways of summarising The Central Object.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: arch1		</title>
		<link>https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-74988</link>

		<dc:creator><![CDATA[arch1]]></dc:creator>
		<pubDate>Wed, 23 Dec 2015 00:02:43 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20209#comment-74988</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-74985&quot;&gt;John Baez&lt;/a&gt;.

Thank you John, I will look for that.

(I just realized btw that my question as phrased is a hybrid one, since a presumably small-but-important subset of the genetic info which is in principle available at a given time exists in already-dead organisms.

So to answer the Q, one must consider not only &#039;evolutionary decay&#039; - the generation-over-generation loss of information concerning the biosphere&#039;s phylogenetic state at a given earlier time due to evolutionary change - but also &#039;physical decay&#039; - the degradation of genetic info in dead organisms.)]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2015/12/21/operads-and-phylogenetic-trees/#comment-74985">John Baez</a>.</p>
<p>Thank you John, I will look for that.</p>
<p>(I just realized btw that my question as phrased is a hybrid one, since a presumably small-but-important subset of the genetic info which is in principle available at a given time exists in already-dead organisms.</p>
<p>So to answer the Q, one must consider not only &#8216;evolutionary decay&#8217; &#8211; the generation-over-generation loss of information concerning the biosphere&#8217;s phylogenetic state at a given earlier time due to evolutionary change &#8211; but also &#8216;physical decay&#8217; &#8211; the degradation of genetic info in dead organisms.)</p>
]]></content:encoded>
		
			</item>
	</channel>
</rss>
