<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#" xmlns:media="http://search.yahoo.com/mrss/"
	
	>
<channel>
	<title>
	Comments on: Biology and the Pi-Calculus	</title>
	<atom:link href="https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/feed/" rel="self" type="application/rss+xml" />
	<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/</link>
	<description></description>
	<lastBuildDate>Tue, 12 Jan 2016 13:07:48 +0000</lastBuildDate>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>http://wordpress.com/</generator>
	<item>
		<title>
		By: Torbjörn Larsson		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-76152</link>

		<dc:creator><![CDATA[Torbjörn Larsson]]></dc:creator>
		<pubDate>Tue, 12 Jan 2016 13:07:48 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-76152</guid>

					<description><![CDATA[&lt;blockquote&gt;
  a minimal set of genes required for the survival of a bacterial cell. If we knew this, it would help us make a guess for the genome of the Last Universal Ancestor.
&lt;/blockquote&gt;

It probably wouldn&#039;t help since last ancestors are routinely found by phylogenetic methods to be surprisingly complex. (E.g. LUCA, LECA, ancestor metazoan, et cetera.)

I am reviewing the latest Astrobiology meeting, and a popular description of life was as an opportunistic geological process. Gisser Martin discuss environmental production of biochemicals that are produced in modern cells, and besides amino acids it seems gluconeogenesis/glycolysis was environmental in Hadean oceans. (See e.g. Keller et al on that.) And so were inorganic cell compartments (pores in vents).

Taking that to the extreme, Woese&#039;s method seems to be much more informative. (I.e. to study the phylogenetics of the ribosome, which today dig deep before the LUCA to the RNA/protein ancestor lineage; see Petrov et al on that, together with the recent uncovered set of ancestral polypeptides.)

A fun observation is that while LUCA likely was a vent occupant, the evolution of chemiosmosis seems to be possible only on the surface of alkaline hydrothermal vents and the heat stress protein root matches in temperature, the above indicates that the whole lineage evolved there. Simplicity may yet again be a good guess.

Since comments on LUCA age abound, I note that recent finds may also put other theories in tension, LUCA may have evolved at neck breaking pace by repetitive accretion as both the ribosome and ancestral polypeptide finds suggest. A putative &#062; 4.1 billion year old fossil* was recently described after painstaking work on old zircons. The find needs to be repeated and better constrained by context as it is a microfossil. But it fits nicely with the &#062; 4.3 billion year habitable ocean (see Valley&#039;s latest review on zircon data) and the &#062; 4.2 billion year old LUCA phylogenetic date of TimeTree (from 3 independent papers!). The carbon isotope ratios was the typical of photosynthesis, set by RuBisCO of the Calvin cycle. That general type of metabolism evolved in the bacteria clade and not in archaea, i.e. it could be dating after the first recognizable split from the LUCA population.

*Actually two fossils nicely encapsulated in one zircon, if we count as paleontologists do.]]></description>
			<content:encoded><![CDATA[<blockquote><p>
  a minimal set of genes required for the survival of a bacterial cell. If we knew this, it would help us make a guess for the genome of the Last Universal Ancestor.
</p></blockquote>
<p>It probably wouldn&#8217;t help since last ancestors are routinely found by phylogenetic methods to be surprisingly complex. (E.g. LUCA, LECA, ancestor metazoan, et cetera.)</p>
<p>I am reviewing the latest Astrobiology meeting, and a popular description of life was as an opportunistic geological process. Gisser Martin discuss environmental production of biochemicals that are produced in modern cells, and besides amino acids it seems gluconeogenesis/glycolysis was environmental in Hadean oceans. (See e.g. Keller et al on that.) And so were inorganic cell compartments (pores in vents).</p>
<p>Taking that to the extreme, Woese&#8217;s method seems to be much more informative. (I.e. to study the phylogenetics of the ribosome, which today dig deep before the LUCA to the RNA/protein ancestor lineage; see Petrov et al on that, together with the recent uncovered set of ancestral polypeptides.)</p>
<p>A fun observation is that while LUCA likely was a vent occupant, the evolution of chemiosmosis seems to be possible only on the surface of alkaline hydrothermal vents and the heat stress protein root matches in temperature, the above indicates that the whole lineage evolved there. Simplicity may yet again be a good guess.</p>
<p>Since comments on LUCA age abound, I note that recent finds may also put other theories in tension, LUCA may have evolved at neck breaking pace by repetitive accretion as both the ribosome and ancestral polypeptide finds suggest. A putative &gt; 4.1 billion year old fossil* was recently described after painstaking work on old zircons. The find needs to be repeated and better constrained by context as it is a microfossil. But it fits nicely with the &gt; 4.3 billion year habitable ocean (see Valley&#8217;s latest review on zircon data) and the &gt; 4.2 billion year old LUCA phylogenetic date of TimeTree (from 3 independent papers!). The carbon isotope ratios was the typical of photosynthesis, set by RuBisCO of the Calvin cycle. That general type of metabolism evolved in the bacteria clade and not in archaea, i.e. it could be dating after the first recognizable split from the LUCA population.</p>
<p>*Actually two fossils nicely encapsulated in one zircon, if we count as paleontologists do.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Glycolysis &#124; Azimuth		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75840</link>

		<dc:creator><![CDATA[Glycolysis &#124; Azimuth]]></dc:creator>
		<pubDate>Fri, 08 Jan 2016 01:00:41 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75840</guid>

					<description><![CDATA[[&#8230;]I’m trying to understand some biology. Being a mathematician I’m less interested in all the complicated details of life on this particular planet than in something a bit more abstract. I want to know ‘the language of life’: the right way to talk about living systems.

Of course, there’s no way to reach this goal without learning a lot of the complicated details. But I might as well be honest and state my goal, since it’s bound to put a strange spin on how I learn and talk about biology.

For example, when I heard people were using the pi-calculus to model a very simple bacterium, I wasn’t eager to know how close their model is to the Last Universal Ancestor, the primordial bug from which we all descend. Even though it’s a fascinating question, it’s not one I can help solve. Instead, I wanted to know if the pi-calculus is really the best language for this kind of model. [&#8230;]]]></description>
			<content:encoded><![CDATA[<p>[&#8230;]I’m trying to understand some biology. Being a mathematician I’m less interested in all the complicated details of life on this particular planet than in something a bit more abstract. I want to know ‘the language of life’: the right way to talk about living systems.</p>
<p>Of course, there’s no way to reach this goal without learning a lot of the complicated details. But I might as well be honest and state my goal, since it’s bound to put a strange spin on how I learn and talk about biology.</p>
<p>For example, when I heard people were using the pi-calculus to model a very simple bacterium, I wasn’t eager to know how close their model is to the Last Universal Ancestor, the primordial bug from which we all descend. Even though it’s a fascinating question, it’s not one I can help solve. Instead, I wanted to know if the pi-calculus is really the best language for this kind of model. [&#8230;]</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75756</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Wed, 06 Jan 2016 19:27:08 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75756</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75751&quot;&gt;leithaus&lt;/a&gt;.

Thanks for the wonderfully detailed comments. Since you note that you&#039;re working with my student Mike Stay, I suppose it&#039;s not giving away a big secret to reveal that &quot;leithaus&quot; is a pseudonym for Greg Meredith.

I wish I knew all the concepts you&#039;re using.  I don&#039;t.  But luckily I get the general drift.

You hinted at this observation, but perhaps not everyone noticed, so I&#039;ll say it out loud: more expressiveness is not always better.  You mentioned one reason: it makes computer security harder:

&lt;blockquote&gt;
  Ambients are so expressive that you can write down leader election (a class of algorithms that’s getting renewed attention what with the growing interest in distributed and decentralized computation on Internet-scale applications) in a single line program. This is simply too expressive to allow as an infrastructure on the Internet at this stage because the trust model for the nodes that run ambient calculus programs would be too high. An attacker could inject code of enormous expressivity into a node and it’s halting hard to say that it doesn’t do unexpectedly bad things
&lt;/blockquote&gt;

But the underlying problem here, that you can&#039;t easily predict the behavior of programs written in highly expressive languages, is also a problem for people wanting to understand models of systems, e.g. in biology.  In modelling systems, I generally want to use the least expressive language necessary for a given task---so I can understand what&#039;s going on in the most detail.

Petri nets are great for understanding a single well-mixed container of chemicals, and because they&#039;re rather inexpressive we can prove lots of great theorems about them---as long as we don&#039;t include fancy features like &#039;inhibitor arcs&#039;.

Supplementing them with extra features is necessary for understanding full-fledged biochemistry, where membranes and &#039;containers&#039; play a big role.   I want to slowly start exploring these extra features, a bit at a time.  But extra features that I don&#039;t actually need will just be confusing.   So you&#039;re making me feel confirmed in what I already thought, that that pi-calculus is not quite right for what I want.  The ambient calculus feels closer, but it feels complicated.  For its mathematical elegance, I&#039;m really attracted to this:

&#8226; Nicolas Ouray and Gordon Plotkin, &lt;a href=&quot;http://homepages.inf.ed.ac.uk/gdp/publications/Multilevel.pdf&quot; rel=&quot;nofollow&quot;&gt;Multi-level modelling via stochastic multi-level multiset rewriting&lt;/a&gt;.

But I&#039;m just getting started...]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75751">leithaus</a>.</p>
<p>Thanks for the wonderfully detailed comments. Since you note that you&#8217;re working with my student Mike Stay, I suppose it&#8217;s not giving away a big secret to reveal that &#8220;leithaus&#8221; is a pseudonym for Greg Meredith.</p>
<p>I wish I knew all the concepts you&#8217;re using.  I don&#8217;t.  But luckily I get the general drift.</p>
<p>You hinted at this observation, but perhaps not everyone noticed, so I&#8217;ll say it out loud: more expressiveness is not always better.  You mentioned one reason: it makes computer security harder:</p>
<blockquote><p>
  Ambients are so expressive that you can write down leader election (a class of algorithms that’s getting renewed attention what with the growing interest in distributed and decentralized computation on Internet-scale applications) in a single line program. This is simply too expressive to allow as an infrastructure on the Internet at this stage because the trust model for the nodes that run ambient calculus programs would be too high. An attacker could inject code of enormous expressivity into a node and it’s halting hard to say that it doesn’t do unexpectedly bad things
</p></blockquote>
<p>But the underlying problem here, that you can&#8217;t easily predict the behavior of programs written in highly expressive languages, is also a problem for people wanting to understand models of systems, e.g. in biology.  In modelling systems, I generally want to use the least expressive language necessary for a given task&#8212;so I can understand what&#8217;s going on in the most detail.</p>
<p>Petri nets are great for understanding a single well-mixed container of chemicals, and because they&#8217;re rather inexpressive we can prove lots of great theorems about them&#8212;as long as we don&#8217;t include fancy features like &#8216;inhibitor arcs&#8217;.</p>
<p>Supplementing them with extra features is necessary for understanding full-fledged biochemistry, where membranes and &#8216;containers&#8217; play a big role.   I want to slowly start exploring these extra features, a bit at a time.  But extra features that I don&#8217;t actually need will just be confusing.   So you&#8217;re making me feel confirmed in what I already thought, that that pi-calculus is not quite right for what I want.  The ambient calculus feels closer, but it feels complicated.  For its mathematical elegance, I&#8217;m really attracted to this:</p>
<p>&bull; Nicolas Ouray and Gordon Plotkin, <a href="http://homepages.inf.ed.ac.uk/gdp/publications/Multilevel.pdf" rel="nofollow">Multi-level modelling via stochastic multi-level multiset rewriting</a>.</p>
<p>But I&#8217;m just getting started&#8230;</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Graham		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75754</link>

		<dc:creator><![CDATA[Graham]]></dc:creator>
		<pubDate>Wed, 06 Jan 2016 17:43:29 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75754</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75570&quot;&gt;John Baez&lt;/a&gt;.

The earliest life is a very different thing from the LAST universal common ancestor. A billion years may have separated the first self-replicator from the LUCA, including the RNA world (https://en.wikipedia.org/wiki/RNA_world).]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75570">John Baez</a>.</p>
<p>The earliest life is a very different thing from the LAST universal common ancestor. A billion years may have separated the first self-replicator from the LUCA, including the RNA world (<a href="https://en.wikipedia.org/wiki/RNA_world" rel="nofollow ugc">https://en.wikipedia.org/wiki/RNA_world</a>).</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: leithaus		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75751</link>

		<dc:creator><![CDATA[leithaus]]></dc:creator>
		<pubDate>Wed, 06 Jan 2016 16:36:04 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75751</guid>

					<description><![CDATA[Regarding Petri nets and expressiveness, there are competing understandings of expressiveness. The older notion was related to functionally complete, aka Turing complete. Can i say anything in this formalism that i can say in lambda calculus or Turing machines? Iirc, Petri nets with inhibitor arcs are functionally complete.

More modern notions of expressiveness stemming from investigations of programming language semantics and the relationship of programming language semantics to category theory are much more nuanced. They get at how well a particular language expresses a particular notion (typically captured in yet another language!). In this domain the moral equivalent of functoriality plays a big role.

Let me illustrate with an example. There are several fragments and variations of the π-calculus. One of them is the asynchronous (no continuations after output), choice-free (no summation construct) π-calculus. This is provably Turing complete (because you can give a rather elementary encoding of the lambda calculus into it). However, if you compare it to another Turing complete fragment, the synchronous (output guards on continuations) π-calculus  with mixed choice (summations that involve both input and output guards), then you can see an expressiveness gap.

Catuscia Palamidessi shows that there is no translation from the latter fragment (synchronous with mixed sums) to the former (asynchronous with no sums) that preserves parallel composition. That is, if [&#124; - &#124;] is our translation function, then insisting that

[&#124; P&#124;Q &#124;] = [&#124; P &#124;] &#124; [&#124; Q &#124;]

exposes an expressiveness gap between the two calculi because no such translation exists. We can argue whether this is a reasonable requirement. But it is still a nice way to separate the two calculi.

There&#039;s a gold standard with respect to expressiveness and this is called full abstraction. Suppose we have two calculi, C1 and C2 and a translation between them [&#124; - &#124;] : C1 -&#062; C2. We want the native notion of equivalence in C1 to exactly match the native notion of equivalence in C2. For example, if C1 is the lambda calculus, then we might use contextual or Morris-style equivalence as the native notion of equivalence, or we might use applicative bisimulation. Meanwhile, if C2 is the π-calculus we might use barbed bisimulation as the native notion of equivalence. Writing ∼ for the equivalence in C1 and ≈ for the equivalence in C2 we can write down what we mean by full abstraction:

[&#124; - &#124;] is fully abstract just when P ∼ Q iff [&#124; P &#124;] ≈ [&#124; Q &#124;].

The content of this definition is that whenever P is substitutable for Q in some computation in C1, then [&#124; P &#124;] is substitutable for [&#124; Q &#124;] in C2. This is a demanding correspondence between the formalisms.

Having laid out these more nuanced notions of expressiveness it becomes possible to get a more nuanced view of when to choose certain formalisms over others when modeling certain kinds of phenomena.

For example, if you compare two mobile process calculi that have been widely used in modeling biological phenomena, namely Milner&#039;s π-calculus with Cardelli and Gordon&#039;s ambient calculus, then you can immediately see practical differences that make a difference. First, as Luca is fond of pointing out, no one has found a fully abstract encoding of the ambient calculus into the π-calculus. So, on the face of it, there appears to be an expressiveness gap.

In the biological domain we are often dealing with phenomena related to membranes. The ambient calculus seems tailor-made for modeling such phenomena, while there&#039;s a kind of conceptual gap in the π-calculus for modeling this kind of thing.

In connection with this Mike Stay and i have been working on results that might be summarized via the slogan &quot;logic is a distributive law&quot; that fits very nicely with our higher-category semantics for the π-calculus. Briefly, if you think of the realizability programme as interpreting formulae in terms of the collections of computations that realize them, then you can interpret a logic as weaving together 3 data: a notion of collecting computations (list, set, bag, etc) which is always modeled as a kind of monad; a notion of computation (lambda, π-calculus, ambient calculus, etc) which turns out to also be modeled in terms of a monad (for the free term algebra); a way of distributing term formation over collections into collections of terms (this turns out to be a distributive law).

Thinking about this idea from the perspective of comparing ambients to π-calculus i have come to suspect that they are talking on completely different levels. i believe that a proper encoding of ambient terms is as modal formulae interpreted over π-calculus terms. This stems from the intuition that an ambient, aka a membrane, identifies a particular kind of collection semantics (everything that&#039;s inside the membrane is collected by the membrane), and the fact that containment in an ambient membrane is dynamic is what motivates the modal aspect of the formulae that interpret ambient agents.

So, this suggests both practically and theoretically why these calculi are of different expressiveness.

Notice that this kind of thing goes the other way. Ambients are so expressive that you can write down leader election (a class of algorithms that&#039;s getting renewed attention what with the growing interest in distributed and decentralized computation on Internet-scale applications) in a single line program. This is simply too expressive to allow as an infrastructure on the Internet at this stage because the trust model for the nodes that run ambient calculus programs would be too high. An attacker could inject code of enormous expressivity into a node and it&#039;s halting hard to say that it doesn&#039;t do unexpectedly bad things. The π-calculus, on the other hand, enjoys a nice result that says you can always encode code-shipping computation into channel-exchanging computation. This makes it much more suitable for today&#039;s Internet with today&#039;s trust model considerations.

To complete the picture and return to the topic of the relationship between Petri nets and π-calculus, Petri nets have a static data flow topology. It can&#039;t change as a result of the computation. The π-calculus has a dynamic communication topology more generally. Which computations are connected to which changes as a result of computation. This corresponds very closely with the way computational activities happen on the Internet. First i connect with amazon.com, but very soon the AMZN servers generate a fresh URL that reflects our specific interaction and during the course of that interaction will generate several more. It also corresponds to what happens in biological interactions where cells, and higher level organisms, are in more or less constant motion. A predator runs after some prey who runs toward a group of more appealing prey of a different species. The communication topology changes. A single cell organism with a flagellum will follow a trail of sugar and move away from a region of poison. The communication topology changes.

It&#039;s very hard to model these kinds of phenomena in the Petri nets. Of course it can be done because Petri nets (with inhibitor arcs) are Turing complete. However, by the time you&#039;ve built the model you&#039;ve probably build a π-calculus interpreter into your translation into Petri nets.

Finally, the biggest problem with Petri nets can be illustrated with the following example. Using the Priami/Silverman/Shapiro approach to modeling chemical reactions in the π-calculus you can do the following kind of thing. Suppose you have a beaker A with some reagents in it and you write down the π-calculus model of that as [&#124; A &#124;]. Similarly, for beaker B with it&#039;s reagents, you write down the π-calculus model as [&#124; B &#124;]. To write down the π-calculus model of beaker C into which you simply pour the contents of beaker A and B, you need only write [&#124; A &#124;] &#124; [&#124; B &#124;], the semantics of parallel composition does the rest. Now, to the best of my knowledge, you can&#039;t do that with Petri nets. If you have an approach that does that, then you&#039;ve got a really significant result that would be of interest to several communities.]]></description>
			<content:encoded><![CDATA[<p>Regarding Petri nets and expressiveness, there are competing understandings of expressiveness. The older notion was related to functionally complete, aka Turing complete. Can i say anything in this formalism that i can say in lambda calculus or Turing machines? Iirc, Petri nets with inhibitor arcs are functionally complete.</p>
<p>More modern notions of expressiveness stemming from investigations of programming language semantics and the relationship of programming language semantics to category theory are much more nuanced. They get at how well a particular language expresses a particular notion (typically captured in yet another language!). In this domain the moral equivalent of functoriality plays a big role.</p>
<p>Let me illustrate with an example. There are several fragments and variations of the π-calculus. One of them is the asynchronous (no continuations after output), choice-free (no summation construct) π-calculus. This is provably Turing complete (because you can give a rather elementary encoding of the lambda calculus into it). However, if you compare it to another Turing complete fragment, the synchronous (output guards on continuations) π-calculus  with mixed choice (summations that involve both input and output guards), then you can see an expressiveness gap.</p>
<p>Catuscia Palamidessi shows that there is no translation from the latter fragment (synchronous with mixed sums) to the former (asynchronous with no sums) that preserves parallel composition. That is, if [| &#8211; |] is our translation function, then insisting that</p>
<p>[| P|Q |] = [| P |] | [| Q |]</p>
<p>exposes an expressiveness gap between the two calculi because no such translation exists. We can argue whether this is a reasonable requirement. But it is still a nice way to separate the two calculi.</p>
<p>There&#8217;s a gold standard with respect to expressiveness and this is called full abstraction. Suppose we have two calculi, C1 and C2 and a translation between them [| &#8211; |] : C1 -&gt; C2. We want the native notion of equivalence in C1 to exactly match the native notion of equivalence in C2. For example, if C1 is the lambda calculus, then we might use contextual or Morris-style equivalence as the native notion of equivalence, or we might use applicative bisimulation. Meanwhile, if C2 is the π-calculus we might use barbed bisimulation as the native notion of equivalence. Writing ∼ for the equivalence in C1 and ≈ for the equivalence in C2 we can write down what we mean by full abstraction:</p>
<p>[| &#8211; |] is fully abstract just when P ∼ Q iff [| P |] ≈ [| Q |].</p>
<p>The content of this definition is that whenever P is substitutable for Q in some computation in C1, then [| P |] is substitutable for [| Q |] in C2. This is a demanding correspondence between the formalisms.</p>
<p>Having laid out these more nuanced notions of expressiveness it becomes possible to get a more nuanced view of when to choose certain formalisms over others when modeling certain kinds of phenomena.</p>
<p>For example, if you compare two mobile process calculi that have been widely used in modeling biological phenomena, namely Milner&#8217;s π-calculus with Cardelli and Gordon&#8217;s ambient calculus, then you can immediately see practical differences that make a difference. First, as Luca is fond of pointing out, no one has found a fully abstract encoding of the ambient calculus into the π-calculus. So, on the face of it, there appears to be an expressiveness gap.</p>
<p>In the biological domain we are often dealing with phenomena related to membranes. The ambient calculus seems tailor-made for modeling such phenomena, while there&#8217;s a kind of conceptual gap in the π-calculus for modeling this kind of thing.</p>
<p>In connection with this Mike Stay and i have been working on results that might be summarized via the slogan &#8220;logic is a distributive law&#8221; that fits very nicely with our higher-category semantics for the π-calculus. Briefly, if you think of the realizability programme as interpreting formulae in terms of the collections of computations that realize them, then you can interpret a logic as weaving together 3 data: a notion of collecting computations (list, set, bag, etc) which is always modeled as a kind of monad; a notion of computation (lambda, π-calculus, ambient calculus, etc) which turns out to also be modeled in terms of a monad (for the free term algebra); a way of distributing term formation over collections into collections of terms (this turns out to be a distributive law).</p>
<p>Thinking about this idea from the perspective of comparing ambients to π-calculus i have come to suspect that they are talking on completely different levels. i believe that a proper encoding of ambient terms is as modal formulae interpreted over π-calculus terms. This stems from the intuition that an ambient, aka a membrane, identifies a particular kind of collection semantics (everything that&#8217;s inside the membrane is collected by the membrane), and the fact that containment in an ambient membrane is dynamic is what motivates the modal aspect of the formulae that interpret ambient agents.</p>
<p>So, this suggests both practically and theoretically why these calculi are of different expressiveness.</p>
<p>Notice that this kind of thing goes the other way. Ambients are so expressive that you can write down leader election (a class of algorithms that&#8217;s getting renewed attention what with the growing interest in distributed and decentralized computation on Internet-scale applications) in a single line program. This is simply too expressive to allow as an infrastructure on the Internet at this stage because the trust model for the nodes that run ambient calculus programs would be too high. An attacker could inject code of enormous expressivity into a node and it&#8217;s halting hard to say that it doesn&#8217;t do unexpectedly bad things. The π-calculus, on the other hand, enjoys a nice result that says you can always encode code-shipping computation into channel-exchanging computation. This makes it much more suitable for today&#8217;s Internet with today&#8217;s trust model considerations.</p>
<p>To complete the picture and return to the topic of the relationship between Petri nets and π-calculus, Petri nets have a static data flow topology. It can&#8217;t change as a result of the computation. The π-calculus has a dynamic communication topology more generally. Which computations are connected to which changes as a result of computation. This corresponds very closely with the way computational activities happen on the Internet. First i connect with amazon.com, but very soon the AMZN servers generate a fresh URL that reflects our specific interaction and during the course of that interaction will generate several more. It also corresponds to what happens in biological interactions where cells, and higher level organisms, are in more or less constant motion. A predator runs after some prey who runs toward a group of more appealing prey of a different species. The communication topology changes. A single cell organism with a flagellum will follow a trail of sugar and move away from a region of poison. The communication topology changes.</p>
<p>It&#8217;s very hard to model these kinds of phenomena in the Petri nets. Of course it can be done because Petri nets (with inhibitor arcs) are Turing complete. However, by the time you&#8217;ve built the model you&#8217;ve probably build a π-calculus interpreter into your translation into Petri nets.</p>
<p>Finally, the biggest problem with Petri nets can be illustrated with the following example. Using the Priami/Silverman/Shapiro approach to modeling chemical reactions in the π-calculus you can do the following kind of thing. Suppose you have a beaker A with some reagents in it and you write down the π-calculus model of that as [| A |]. Similarly, for beaker B with it&#8217;s reagents, you write down the π-calculus model as [| B |]. To write down the π-calculus model of beaker C into which you simply pour the contents of beaker A and B, you need only write [| A |] | [| B |], the semantics of parallel composition does the rest. Now, to the best of my knowledge, you can&#8217;t do that with Petri nets. If you have an approach that does that, then you&#8217;ve got a really significant result that would be of interest to several communities.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: domenico		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75745</link>

		<dc:creator><![CDATA[domenico]]></dc:creator>
		<pubDate>Wed, 06 Jan 2016 14:42:39 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75745</guid>

					<description><![CDATA[I don&#039;t see much difference between a chain of chemical reaction and a human thought, through a chemical action potential.
It can be that the cell signaling and g-proteins can forming a though in a single cell, like a primordial brain; so that it could be possible to write a primordial theory for the chain of chemical reactions to obtain a first conscious program.
The last universal ancestor could be a first self-replicating rna that it is connected with an helical capsid, so that in a right - with complex molecules – environment the evolution can be possible, with a great and stable Helmholtz free energy reservoir (like a hydrothermal vent) that could generate these complex molecules.
It is interesting that some researchers try to simulate some complex biological behaviour using all branches of knowledge.]]></description>
			<content:encoded><![CDATA[<p>I don&#8217;t see much difference between a chain of chemical reaction and a human thought, through a chemical action potential.<br />
It can be that the cell signaling and g-proteins can forming a though in a single cell, like a primordial brain; so that it could be possible to write a primordial theory for the chain of chemical reactions to obtain a first conscious program.<br />
The last universal ancestor could be a first self-replicating rna that it is connected with an helical capsid, so that in a right &#8211; with complex molecules – environment the evolution can be possible, with a great and stable Helmholtz free energy reservoir (like a hydrothermal vent) that could generate these complex molecules.<br />
It is interesting that some researchers try to simulate some complex biological behaviour using all branches of knowledge.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Marc Harper		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75639</link>

		<dc:creator><![CDATA[Marc Harper]]></dc:creator>
		<pubDate>Mon, 04 Jan 2016 23:16:48 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75639</guid>

					<description><![CDATA[Regarding minimal sets of genes, it&#039;s crucial to identify the environment. For example, Spiegelman&#039;s monster is an RNA strand selected for minimal length in a solution that contained the proteins necessary for replication, and so those proteins did not need to be coded for in the final/minimal genome, and indeed they were lost after many generations of evolution. [1]

For bacteria, if the environment is e.g. a Petri dish, then the nutrient medium is crucial. If glucose is the only carbon source in the medium then the cell does not need the genes for other utilizing other carbon sources (unless, of course, those genes are critical for other metabolic pathways). We don&#039;t really know what the environment of the last universal ancestor was, but knowing some of its genes would shed some light on its environment -- it&#039;s an intringuing question biologically. Of course some information may have been irrevocably lost since there have been major changes to the global enviroment (reductive to oxidative being a big one), but perhaps that information is preserved in extremophiles.

I would guess that a minimal set of genes for say E. coli could be arrived at empirically by knocking out genes in successive generations perhaps pathway by pathway, using known functional association databases to avoid genes that are involved in multiple pathways, and allowing other genes to atrophy by mutation. It would be tedious work but within current technological reach. This could be repeated for other organisms (preferrably distintly related) and the intersection of the minimal genes sets would likely have to been in the LUA (more precisely, genes that are ancestors of the common genes). I&#039;m imagining something much like the inference of extinct ancestor languages in linguistics.

[1] https://en.wikipedia.org/wiki/Spiegelman&#039;s_Monster]]></description>
			<content:encoded><![CDATA[<p>Regarding minimal sets of genes, it&#8217;s crucial to identify the environment. For example, Spiegelman&#8217;s monster is an RNA strand selected for minimal length in a solution that contained the proteins necessary for replication, and so those proteins did not need to be coded for in the final/minimal genome, and indeed they were lost after many generations of evolution. [1]</p>
<p>For bacteria, if the environment is e.g. a Petri dish, then the nutrient medium is crucial. If glucose is the only carbon source in the medium then the cell does not need the genes for other utilizing other carbon sources (unless, of course, those genes are critical for other metabolic pathways). We don&#8217;t really know what the environment of the last universal ancestor was, but knowing some of its genes would shed some light on its environment &#8212; it&#8217;s an intringuing question biologically. Of course some information may have been irrevocably lost since there have been major changes to the global enviroment (reductive to oxidative being a big one), but perhaps that information is preserved in extremophiles.</p>
<p>I would guess that a minimal set of genes for say E. coli could be arrived at empirically by knocking out genes in successive generations perhaps pathway by pathway, using known functional association databases to avoid genes that are involved in multiple pathways, and allowing other genes to atrophy by mutation. It would be tedious work but within current technological reach. This could be repeated for other organisms (preferrably distintly related) and the intersection of the minimal genes sets would likely have to been in the LUA (more precisely, genes that are ancestors of the common genes). I&#8217;m imagining something much like the inference of extinct ancestor languages in linguistics.</p>
<p>[1] <a href="https://en.wikipedia.org/wiki/Spiegelman&#039;s_Monster" rel="nofollow ugc">https://en.wikipedia.org/wiki/Spiegelman&#039;s_Monster</a></p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75622</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Mon, 04 Jan 2016 18:09:48 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75622</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75575&quot;&gt;Mike Stay&lt;/a&gt;.

Thanks!   I know about some of those papers.  I want to get into this stuff more deeply now.

Corrado Priami introduced the stochastic pi-calculus and applied it to biology here:

&#8226; C. Priami, Stochastic pi-calculus, &lt;em&gt;The Computer Journal&lt;/em&gt; &lt;strong&gt;38&lt;/strong&gt; (1995), 578&#8211;589.

&#8226; C. Priami, A. Regev, E. Shapiro, W. Silverman, Application of a stochastic name-passing calculus to representation and simulation of molecular processes, &lt;em&gt;Information Processing Letters&lt;/em&gt; &lt;strong&gt;80&lt;/strong&gt; (2001), 25.

Also, COSBI (www.cosbi.eu) developed a set of languages to model biological systems which are related to the pi-calculus (e.g., beta-binders, BlenX, L languages). Most of them are implemented and freely available online (http://www.cosbi.eu/research/prototypes  and also  http://www.cosbi.eu/research/cosbi-lab). Some relevant papers are here:

&#8226; C. Priami and P. Quaglia, Beta binders for biological interactions. &lt;em&gt;Computational Methods in Systems Biology&lt;/em&gt; (2005), 20&#8211;33.

&#8226; L. Dematte, C. Priami, and A. Romanel, The Beta Workbench: a computational tool to study the dynamics of biological systems, &lt;em&gt;Briefings in Bioinformatics&lt;/em&gt; &lt;strong&gt;9&lt;/strong&gt; (2008), 437&#8211;449.

&#8226; R. Gostner, B. Baldacci, M.J. Morine and C. Priami, Graphical modelling tools for systems biology, &lt;em&gt;ACM Computing Surveys&lt;/em&gt; &lt;strong&gt;47&lt;/strong&gt; (2014).

Lot of details on language-based models of biology can be found in this book:

&#8226; C. Priami and M.J. Morine, &lt;em&gt;Analysis of Biological Systems&lt;/em&gt;, Imperial College Press, 2015.

I need to get this and read it!]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75575">Mike Stay</a>.</p>
<p>Thanks!   I know about some of those papers.  I want to get into this stuff more deeply now.</p>
<p>Corrado Priami introduced the stochastic pi-calculus and applied it to biology here:</p>
<p>&bull; C. Priami, Stochastic pi-calculus, <em>The Computer Journal</em> <strong>38</strong> (1995), 578&ndash;589.</p>
<p>&bull; C. Priami, A. Regev, E. Shapiro, W. Silverman, Application of a stochastic name-passing calculus to representation and simulation of molecular processes, <em>Information Processing Letters</em> <strong>80</strong> (2001), 25.</p>
<p>Also, COSBI (www.cosbi.eu) developed a set of languages to model biological systems which are related to the pi-calculus (e.g., beta-binders, BlenX, L languages). Most of them are implemented and freely available online (<a href="http://www.cosbi.eu/research/prototypes" rel="nofollow ugc">http://www.cosbi.eu/research/prototypes</a>  and also  <a href="http://www.cosbi.eu/research/cosbi-lab" rel="nofollow ugc">http://www.cosbi.eu/research/cosbi-lab</a>). Some relevant papers are here:</p>
<p>&bull; C. Priami and P. Quaglia, Beta binders for biological interactions. <em>Computational Methods in Systems Biology</em> (2005), 20&ndash;33.</p>
<p>&bull; L. Dematte, C. Priami, and A. Romanel, The Beta Workbench: a computational tool to study the dynamics of biological systems, <em>Briefings in Bioinformatics</em> <strong>9</strong> (2008), 437&ndash;449.</p>
<p>&bull; R. Gostner, B. Baldacci, M.J. Morine and C. Priami, Graphical modelling tools for systems biology, <em>ACM Computing Surveys</em> <strong>47</strong> (2014).</p>
<p>Lot of details on language-based models of biology can be found in this book:</p>
<p>&bull; C. Priami and M.J. Morine, <em>Analysis of Biological Systems</em>, Imperial College Press, 2015.</p>
<p>I need to get this and read it!</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: John Baez		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75583</link>

		<dc:creator><![CDATA[John Baez]]></dc:creator>
		<pubDate>Sun, 03 Jan 2016 00:06:31 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75583</guid>

					<description><![CDATA[In reply to &lt;a href=&quot;https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75571&quot;&gt;John Baez&lt;/a&gt;.

Martin wrote:

&lt;blockquote&gt;
  1) The question of autopoiesis: Are the enzymes/channels produced by other enzymes/channels?
&lt;/blockquote&gt;

Unfortunately they don&#039;t seem to say!  First they simulated their competitor&#039;s cell, the one with the claimed &quot;minimal gene set&quot;.  It crashed and died quite soon:

&lt;img width=&quot;380&quot; src=&quot;http://math.ucr.edu/home/baez/chemical/minimal_gene_set_atp_graph.png&quot; /&gt;

Their own model cell lasted longer.  But I don&#039;t think they got it to reproduce!  So it could be like simulating a guy but leaving out his testes.  There&#039;s &lt;a href=&quot;http://www.nytimes.com/2012/07/21/science/in-a-first-an-entire-organism-is-simulated-by-software.html&quot; rel=&quot;nofollow&quot;&gt;another experiment&lt;/a&gt; where people claim to have simulated the whole life cycle of a bacterium, including reproduction.  But that was a much bigger product:

&lt;blockquote&gt;
  The simulation of the complete life cycle of the pathogen, &lt;em&gt;Mycoplasma genitalium&lt;/em&gt;, was presented on Friday &lt;a href=&quot;http://files.figshare.com/95033/dataS1.pdf&quot; rel=&quot;nofollow&quot;&gt;in the journal &lt;i&gt;Cell&lt;/i&gt;&lt;/a&gt;. The scientists called it a “first draft” but added that the effort was the first time an entire organism had been modeled in such detail — in this case, all of its 525 genes.
  
  The simulation, which runs on a cluster of 128 computers, models the complete life span of the cell at the molecular level, charting the interactions of 28 categories of molecules — including DNA, RNA, proteins and small molecules known as metabolites, which are generated by cell processes.
  
  [...]
  
  For the new computer simulation, the researchers had the advantage of extensive scientific literature on the bacterium. They were able to use data taken from more than 900 scientific papers to validate the accuracy of their software model.
  
  Still, they said, the model of the simplest biological system was pushing the limits of their computers.
  
  “Right now, running a simulation for a single cell to divide only one time takes around 10 hours and generates half a gigabyte of data,” Dr. Covert wrote. “I find this fact completely fascinating, because I don’t know that anyone has ever asked how much data a living thing truly holds. We often think of the DNA as the storage medium, but clearly there is more to it than that.”
&lt;/blockquote&gt;

Martin wrote:

&lt;blockquote&gt;
2)  They write: “ViCe has no pathways for amino acid synthesis.” Do amino acids occur in abiotic nature? I guess not. Then ViCe would be far away from the primordial ancestral cell.
&lt;/blockquote&gt;

There actually are some amino acids in abiotic nature: there&#039;s even &lt;a href=&quot;http://physicsworld.com/cws/article/news/2003/aug/11/amino-acid-detected-in-space&quot; rel=&quot;nofollow&quot;&gt;glycine in interstellar clouds&lt;/a&gt; and &lt;a href=&quot;http://www.nasa.gov/mission_pages/stardust/news/stardust_amino_acid.html&quot; rel=&quot;nofollow&quot;&gt;comets&lt;/a&gt;.  But the cell being simulated here is not an attempt to reconstruct one of the first organisms: it&#039;s an attempt to reconstruct the &lt;em&gt;last&lt;/em&gt; organism that&#039;s an ancestor to all living today.  So, it would be the result of at least hundreds of millions of years of evolution, and it would probably live in an environment where &#039;primordial&#039; supplies of chemicals had already been largely eaten.]]></description>
			<content:encoded><![CDATA[<p>In reply to <a href="https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75571">John Baez</a>.</p>
<p>Martin wrote:</p>
<blockquote><p>
  1) The question of autopoiesis: Are the enzymes/channels produced by other enzymes/channels?
</p></blockquote>
<p>Unfortunately they don&#8217;t seem to say!  First they simulated their competitor&#8217;s cell, the one with the claimed &#8220;minimal gene set&#8221;.  It crashed and died quite soon:</p>
<p><img width="380" src="https://i2.wp.com/math.ucr.edu/home/baez/chemical/minimal_gene_set_atp_graph.png" /></p>
<p>Their own model cell lasted longer.  But I don&#8217;t think they got it to reproduce!  So it could be like simulating a guy but leaving out his testes.  There&#8217;s <a href="http://www.nytimes.com/2012/07/21/science/in-a-first-an-entire-organism-is-simulated-by-software.html" rel="nofollow">another experiment</a> where people claim to have simulated the whole life cycle of a bacterium, including reproduction.  But that was a much bigger product:</p>
<blockquote><p>
  The simulation of the complete life cycle of the pathogen, <em>Mycoplasma genitalium</em>, was presented on Friday <a href="http://files.figshare.com/95033/dataS1.pdf" rel="nofollow">in the journal <i>Cell</i></a>. The scientists called it a “first draft” but added that the effort was the first time an entire organism had been modeled in such detail — in this case, all of its 525 genes.</p>
<p>  The simulation, which runs on a cluster of 128 computers, models the complete life span of the cell at the molecular level, charting the interactions of 28 categories of molecules — including DNA, RNA, proteins and small molecules known as metabolites, which are generated by cell processes.</p>
<p>  [&#8230;]</p>
<p>  For the new computer simulation, the researchers had the advantage of extensive scientific literature on the bacterium. They were able to use data taken from more than 900 scientific papers to validate the accuracy of their software model.</p>
<p>  Still, they said, the model of the simplest biological system was pushing the limits of their computers.</p>
<p>  “Right now, running a simulation for a single cell to divide only one time takes around 10 hours and generates half a gigabyte of data,” Dr. Covert wrote. “I find this fact completely fascinating, because I don’t know that anyone has ever asked how much data a living thing truly holds. We often think of the DNA as the storage medium, but clearly there is more to it than that.”
</p></blockquote>
<p>Martin wrote:</p>
<blockquote><p>
2)  They write: “ViCe has no pathways for amino acid synthesis.” Do amino acids occur in abiotic nature? I guess not. Then ViCe would be far away from the primordial ancestral cell.
</p></blockquote>
<p>There actually are some amino acids in abiotic nature: there&#8217;s even <a href="http://physicsworld.com/cws/article/news/2003/aug/11/amino-acid-detected-in-space" rel="nofollow">glycine in interstellar clouds</a> and <a href="http://www.nasa.gov/mission_pages/stardust/news/stardust_amino_acid.html" rel="nofollow">comets</a>.  But the cell being simulated here is not an attempt to reconstruct one of the first organisms: it&#8217;s an attempt to reconstruct the <em>last</em> organism that&#8217;s an ancestor to all living today.  So, it would be the result of at least hundreds of millions of years of evolution, and it would probably live in an environment where &#8216;primordial&#8217; supplies of chemicals had already been largely eaten.</p>
]]></content:encoded>
		
			</item>
		<item>
		<title>
		By: Mike Stay		</title>
		<link>https://johncarlosbaez.wordpress.com/2016/01/02/biology-and-the-pi-calculus/#comment-75575</link>

		<dc:creator><![CDATA[Mike Stay]]></dc:creator>
		<pubDate>Sat, 02 Jan 2016 21:59:24 +0000</pubDate>
		<guid isPermaLink="false">http://johncarlosbaez.wordpress.com/?p=20511#comment-75575</guid>

					<description><![CDATA[Luca Cardelli has done a lot of stuff with simulating cells with pi calculus: 
http://lucacardelli.name/biocomputing.htm]]></description>
			<content:encoded><![CDATA[<p>Luca Cardelli has done a lot of stuff with simulating cells with pi calculus:<br />
<a href="http://lucacardelli.name/biocomputing.htm" rel="nofollow ugc">http://lucacardelli.name/biocomputing.htm</a></p>
]]></content:encoded>
		
			</item>
	</channel>
</rss>
